### PR TITLE
update rebuildstaging workflow to use external tool

### PR DESCRIPTION
**Pull request by snopoke**
Move the rebuild staging tool and associated utils to https://pypi.org/project/git-build-branch/

### CODE REVIEW

File: requirements/dev-requirements.in [hunk 1]
```diff
@@ -12,6 +12,7 @@ wheel==0.29.0
 transifex-client==0.12.4
 modernize
 gnureadline==6.3.8
+git-branch-builder
```
Comment (millerdev) [inline requirements/dev-requirements.in:15]: nit: git commands are usually verbs or verb phrases. Ex: `git-build-branch`. Is it too late to change that?

**Review by millerdev — APPROVED** [review]

File: requirements/dev-requirements.in (hunk 1 above)
Comment (millerdev) [reply requirements/dev-requirements.in]: :white_check_mark: Thanks!

Comment (dannyroberts) [reply requirements/dev-requirements.in]: Is there still some way to go in the renaming? The pypi entry and the repo https://github.com/dimagi/git-build-branch are renamed, but looking at the code _in_ the repo it looks like the main directory as well as the executable are still called git-branch-builder.

**Review by dannyroberts — COMMENTED** [review]
This is awesome. What's the new dev workflow? Looks like it's an in place refactor so we still use `./scripts/commit-staging` and `./scripts/rebuildstaging`?

File: requirements/dev-requirements.in (hunk 1 above)
Comment (snopoke) [reply requirements/dev-requirements.in]: I forgot to push my changes. The repo is now updated.

**Comment by sravfeyn** [comment]
I have installed the new requirement and tries `./scripts/rebuildstaging`, but it just waits for a long time without any message at 

```
rebuilding staging branch, this might take a while...
Fetching master
```

**Comment by sravfeyn** [comment]
Works okay now.

**Pull request merged**